### DISCUSSION TITLE
GitHub Authentication

### DISCUSSION THREAD

**Discussion by fasmide** [fasmide/remotemoe#14] [Ideas]
So, in light of #12 I've been thinking about how to authorize remote.moe's users without getting too much in the way

One way of doing it could be leveraging GitHubs public key API `https://github.com/<username>.keys` and use these keys for identification of users

At the moment, remotemoe does not use the username for anything, it shouldn't be a problem to have users provide their GitHub username and then lookup their keys on the fly

**Comment by Daviljoe193** [comment]
I use RemoteMoe for my public Colab notebook, so hopefully there's something I can do to be able to be able to continue using RemoteMoe. You know, without effectively leaving the auth credentials out in the open. Currently the notebook is meant to be a completely frictionless and unintimidating way to run an LLM with API access tunneled through RemoteMoe. Needing to add something else for the user to need to change every time before running the notebook kinda goes against my whole attempt to streamline everything.

**Comment by larax12** [comment]
ok

**Reply by Ya1000300** [reply]
.

**Comment by soumyadeep394** [comment]
![cute lady](https://github.com/fasmide/remotemoe/assets/167521122/4e0b3f43-4b80-4e47-806e-650040e0057f)

**Comment by fasmide** [comment]
I've had some abuse complaints about phishing attempts being hosted through remotemoe, so - I'm closing the service down until further notice.

![image](https://github.com/fasmide/remotemoe/assets/1194937/27575fc1-d695-4120-b16b-ecff78642fb3)

**Reply by GiLonga** [reply]
Hi, now that remotemoe is close, what options do I have to utilize AUTOMATIC1111 without a personal GPU? Many thanks for the help!

**Reply by top-master** [reply]
Custom host remotemoe, it's open-source for a reason.
And you don't even need to buy a domain-name, just access said host by its IP address.

I just wish there was a way to allow only a single person to use said custom host, I mean, next thing you will be used for phishing as well.

**Comment by Sararere777** [comment]
Hi

**Comment by Alam-General** [comment]
please bring your authentication as soon as possible. One more request keep authentication process as simple as possible.

**Comment by aditya720p** [comment]
Kindly add some warning page to prevent fraud. U can also add password authentication.

**Comment by addy005** [comment]
add warning page like ngrok

**Comment by soumyadeep394** [comment]
I randomly sign up this website and can't understand  what are you saying
and remember I don't understand English if I do any wrong please object me
by phone call in Hindi language thankyou...

**Comment by ghost** [comment]
how to upgrade virsion i m getting the link bt not open the site

**Comment by addy005** [comment]
lmao its dead now

**Reply by wuwanwuwan** [reply]
[remote.moe (the service) is closed down until a viable authentication scheme is implemented.]

I have set my SSH key. I don't know where is the problem.

**Reply by top-master** [reply]
The SSH key is just for encryption, and for now has nothing to do with authentication, where with "for now" I mean that, remotemoe could add web-form for registering your SSH keys, but then randomly generated keys won't work.

**Comment by munna1234mm** [comment]
[README (1).md](https://github.com/fasmide/remotemoe/files/15286744/README.1.md)

**Comment by Alam-General** [comment]
> So, in light of #12 I've been thinking about how to authorize remote.moe's users without getting too much in the way
> 
> One way of doing it could be leveraging GitHubs public key API `https://github.com/<username>.keys` and use these keys for identification of users
> 
> At the moment, remotemoe does not use the username for anything, it shouldn't be a problem to have users provide their GitHub username and then lookup their keys on the fly

please bring back the remote.moe

**Reply by addy005** [reply]
its gone

**Comment by abubakar4447** [comment]
how to use it now?

**Reply by addy005** [reply]
its dead

**Reply by top-master** [reply]
For now by self-hosting: https://github.com/fasmide/remotemoe/discussions/14#discussioncomment-9409826

Otherwise, wait until authentication support is added ;-)

**Comment by innu-art** [comment]
remote.moe (the service) is closed down until a viable authentication scheme is implemented

remote.moe is not working.
Connection to remote.moe closed by remote host.
Connection to remote.moe closed.

**Comment by dipdarshan** [comment]
Good to use

**Comment by Mmmmmmmm786** [comment]
Funny video banane ki jarurat hai 

https://github.com/user-attachments/assets/31521316-7be7-4520-8999-ce50161c453a

**Comment by aadityaverma** [comment]
any alternatives ?

**Comment by habla72** [comment]
1.

**Comment by Ivang71** [comment]
Did anyone find any alternatives?

**Comment by Nuneur07** [comment]
Didn't open the page

**Comment by Sudip-1234** [comment]
ggh

**Comment by kk3021891-sys** [comment]
![image_search_1759469167842](https://github.com/user-attachments/assets/1ab4cb3e-5d87-4126-8c0e-c2a2035fcbfd)

**Comment by kk3021891-sys** [comment]
Dance movement

**Reply by kk3021891-sys** [reply]
Red t-shirt this image

**Comment by messageprintadv-netizen** [comment]
> https://github.com/<username>.keys

<img width="1024" height="1536" alt="17" src="https://github.com/user-attachments/assets/3631607a-775e-4e2b-ad29-35e0c3f03b3c" />
